resources/js/Organisms/Form.tsx

```diff
@@ -3,7 +3,7 @@ import React, { useState } from "react";
 interface IFormProps {
```

Comment:
It would be useful in future if we made this accept a generic type parameter (e.g. `IFormProps<TValues>` specifying exactly what shape `IValues` has, rather than it just being any `{[key: string]: any;}`. That way we can enforce that the validation and submit functions also have the same type parameter passed into them.

We don't need to worry about doing this now though, but it might be something to look at in future. I think it will be valuable to help avoid bugs, particularly once we implement the validation and submit functions properly. I think those functions might also need to be passed in as props in future.. Depends on how you are expecting the `Form` component to be used.